Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -545,6 +554,25 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor {
   UPROPERTY(BlueprintAssignable, Category = "Cesium");
   FCompletedLoadTrigger OnTilesetLoaded;
 
+  /**
+   * Use a dithering effect when transitioning between tiles of different LODs.
+   */
+  UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Cesium|Rendering")
+  bool UseLodTransitions = true;
+
+  /**
+   * How long dithered LOD transitions between different tiles should take, in
+   * seconds.
+   *
+   * Only relevant if UseLodTransitions is true.
+   */
+  UPROPERTY(
+      EditAnywhere,
+      BlueprintReadWrite,
+      Category = "Cesium|Rendering",
+      meta = (EditCondition = "UseLodTransitions"))
```

Comment:
```suggestion
      meta = (EditCondition = "UseLodTransitions", EditConditionHides))
```
No need to show this in the panel if dithering is disabled.

Reply:
Awesome idea, hadn't heard about this specifier before!